neurobooth_os/server_stm.py

```diff
@@ -33,18 +31,26 @@
 from neurobooth_os.tasks.wellcome_finish_screens import welcome_screen, finish_screen
 import neurobooth_os.tasks.utils as utl
 from neurobooth_os.tasks.task_importer import get_task_funcs
-from neurobooth_os.logging import make_session_logger
+from neurobooth_os.logging import make_session_logger, make_default_logger
 
 
 def Main():
     os.chdir(neurobooth_os.__path__[0])
-
     sys.stdout = NewStdout("STM", target_node="control", terminal_print=True)
-    s1 = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 
-    # Initialize logging to nothing, will get overwritten during session preparation
-    logger = logging.getLogger('null')
-    logger.addHandler(logging.NullHandler())
+    # Initialize logging to default
+    logger = make_default_logger()
+
+    try:
+        run_stm(logger)
```

Comment:
Is the intent to replace the default logger with the session logger during prepare? That seems reasonable to me - just making sure I understand correctly.

Reply:
yes. For now, at least. Centralizing the logs would be a separate PR